docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -338,7 +319,66 @@ You can also look at the backup files:
    find /tmp/backups
 
 
-If `ansible-playbook` fails, please follow the debug steps in :doc:`network_debug_troubleshooting`.
+If `ansible-playbook` fails, please follow the debug steps in :ref:`network_debug_troubleshooting`.
+
+
+.. _network-agnostic-examples:
+
+Example 2: simplifying playbooks with network agnostic modules
+==============================================================
+
+If you have two or more network platforms in your environment, you can use the network agnostic modules to simplify your playbooks. You can use network agnostic modules such as ``cli_command`` or ``cli_config`` in place of the platform-specific modules such as ``eos_config``, ``ios_config``, and ``junos_config``. This reduces the number of tasks and conditionals you need in your playbooks.
+
+.. note::
+  Network agnostic modules require the ``network_cli`` connection plugin.
```

Comment:
I'd add a link to the connection plugins docs, or directly to the `network_cli` doc.